dev-tools/action-install-fluvio-cluster.sh

```diff
@@ -16,6 +16,12 @@ if [ "$DEVELOPMENT" = "true" ]; then
                 # Install Fluvio Src
                 cargo install --path ./src/cli
                 popd
+        elif [ "$VERSION" = "v0.6.0-alpha.3" ]; then 
```

Comment:
can we avoid hard coding version number? can consumer of this action use git commit or tag for dependency to this script?

Reply:
This version corresponds to the release version, how do you imagine using a ref for the download link?

Reply:
there should be version as optional argument? if option is not specified then use latest

Reply:
Commit 7041481 changes this to now search for a `v` release tag in the version, removing the hardcoded value;

```bash
...
elif [ "$VERSION" == *"v"* ]; then 
...
```